src/cart/crt_iv.c

```diff
@@ -135,6 +136,14 @@ struct crt_ivns_internal {
 	void				*cii_user_priv;
 };
 
+void
+crt_iv_init(crt_init_options_t *ops)
+{
+	if (ops != NULL && ops->cio_max_unexpected_size > 1024)
+		crt_iv_inline_limit = ops->cio_max_expected_size - 1024;
+	D_INFO("max inline buf size is %u\n", crt_iv_inline_limit);
```

Comment:
oh, this seems important information for debugging. since it only show 1 time per xstream, so just put it to D_INFO, which should not flood the console.